test/format/config.h

```diff
@@ -250,6 +250,10 @@ static CONFIG c[] = {
 	  "the number of LSM worker threads",
 	  0x0, 3, 4, 20, &g.c_lsm_worker_threads, NULL },
 
+	{ "memory_page_max",
+	  "maximum size of in-memory pages",
+	  0x0, 1, 10, 128, &g.c_memory_page_max, NULL },
```

Comment:
I expected to see a `CONFIG_APPEND` line in `wts.c` that uses this value: did I miss it?

Reply:
You're correct, I missed that, pushed in b077abf.